src/test/java/com/mael/ttt/BoardTest.java

```diff
@@ -39,7 +47,7 @@ public void resetsBoard() {
     @Test
     public void getsTheEmptyCellIndexes() {
         board.setCell(1, PLAYER);
-        List<Integer> expected = asList(2, 3, 4, 5, 6, 7, 8, 9);
+        List<Integer> expected = getCellIndexesStartingOn(2);
```

Comment:
I mentioned that in a previous PR already, I think it's good that you wanted to make sure that all boards behave the same no matter the size.
But the solution now has a bit too much magic going on and becomes harder to follow and understand.

Hence why I'd like to ask you to rethink and question how much of the behaviour of a 4x 4 is different to a 3x3 board.

Would it be enough to leave the `BoardTests` as it was and add a few more methods that create a 4x4 board explicitly?

Same goes for the `BoardChecker`. Of course, you want to make sure that winning rows and columns can be identified on a 4x4 board. But maybe it's enough to just create a fixed 4x4 and ask the checker if it has a winner or not.